### DISCUSSION TITLE
tengo un problema con mi codigo

### DISCUSSION THREAD

**Discussion by lamakina12** [python-telegram-bot/python-telegram-bot#3944] [Q&A]
tengo este error y no deja correr el codigo este es el codigo

```python
from telegram import Update
from telegram.ext import Updater, CommandHandler, CallbackContext

def start(update: Update, context: CallbackContext) -> None:
    update.message.reply_text('¡Hola! Soy tu bot.')

def main() -> None:
    updater = Updater("TOKEN", use_context=True)

    dispatcher = updater.dispatcher

    dispatcher.add_handler(CommandHandler("start", start))

    updater.start_polling()

    updater.idle()

if __name__ == '__main__':
    main()
```

![photo_2023-10-23_20-32-46](https://github.com/python-telegram-bot/python-telegram-bot/assets/55039766/ebc2d538-47fa-4535-92ff-7ecdd2dda8bd)
![photo_2023-10-23_20-32-46](https://github.com/python-telegram-bot/python-telegram-bot/assets/55039766/2c0d4575-68ab-4fcc-9c8f-73c6bcf8b30a)

**Comment by Bibo-Joshi** [comment]
Hey. You seem to be using a version >=20.0 of python-telegram-bot  but your code is written for version <=13.15. Please either upgrade your code to  v20 by reading the release notes and the transition guide linked below or install a version of PTB that is compatible with your code base.
Please note that we only provide support for the latest stable version and that the library has undergone significant changes in v20.

https://telegra.ph/Release-notes-for-python-telegram-bot-v200a0-05-06
https://github.com/python-telegram-bot/python-telegram-bot/wiki/Transition-guide-to-Version-20.0

Also post tracebacks as monospaced text instead of posting screenshots and kindly use english in this international forum ;)
